packages/reports/tests/dummy/config/deprecation-workflow.js

```diff
@@ -0,0 +1,19 @@
+self.deprecationWorkflow = self.deprecationWorkflow || {};
```

Comment:
Deprecation workflow added to cut down on the warnings as it was causing the log buffer to overflow in travis.